README.md

```diff
@@ -122,3 +157,38 @@ encode_number_precision
 This fork allows encoding of numbers with a `precision` up to 16 decimals (vs. 14 in mpx/lua-cjson).
 
 [Back to TOC](#table-of-contents)
+
+decode_array_with_array_mt
+--------------------------------
+**syntax:** `cjson.decode_array_with_array_mt(enabled)`
+
+**default:** true
```

Comment:
Should we turn this off by default for the best backward compatibility?

Reply:
I am ok with this, but I think the default behavior really is broken, and that backwards compatibility shouldn't suffer if we enable this by default. I'll turn it off anyways just to be safe.